DVR/SessionDownloadTask.swift

```diff
@@ -0,0 +1,44 @@
+class SessionDownloadTask: NSURLSessionDownloadTask {
+
+    // MARK: - Types
+
+    typealias Completion = (NSURL?, NSURLResponse?, NSError?) -> Void
```

Comment:
Can we yank this `typealias` up to the `Session` or something so we don't have to declare it twice?